runbotics/runbotics-orchestrator-ui/src/src-landing/components/LanguageSwitcher/LanguageSwitcher.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '#src-landing/scss/main';
+@import '#src-landing/components/Typography/Typography.module.scss';
+
+.option {
+    font-family: 'Montserrat';
+    font-size: large;
+    font-weight: 500;
+    padding: 10px;
+    cursor: pointer;
+}
+.selectWrapper {
+    position: relative;
+    width: 40px;
+    aspect-ratio: 1/1;
+    display: flex;
+    border: 2px solid black;
+    border-radius: 0.25em;
+    font-size: 1.25rem;
+    cursor: pointer;
+    background-color: rgba(255, 255, 255, 0.103);
+    align-items: center;
+
+    grid-template-areas: 'select';
+    &::-ms-expand {
+        display: none;
+    }
+    &::after {
+        position: absolute;
+        left: 0;
+        right: 0;
+        margin-left: auto;
+        margin-right: auto;
+        content: '';
+        background: url('/images/flags/globe.svg') no-repeat;
+        width: 30px;
+        height: 30px;
+        background-size: cover;
+    }
+}
+.select {
+    border-radius: 5px;
+    padding: 10px;
+    z-index: 99;
```

Comment:
every z-index is documented in _z-indes.scss. If you need new one then place it there. Maybe you can use existing one +1